pycontrails/core/vector.py

```diff
@@ -1913,29 +2001,115 @@ def vector_to_lon_lat_grid(
 
 
 def _handle_time_column(time: pd.Series) -> pd.Series:
+    """Ensure that pd.Series has compatible Timestamps.
+
+    Parameters
+    ----------
+    time : pd.Series
+        Pandas dataframe column labeled "time".
+
+
+    Returns
+    -------
+    pd.Series
+        Parsed pandas time series.
+
+
```

Comment:
There may be some extra white space here

Reply:
4c022449eba73f91892f25d2123b118335b1ae48